### DISCUSSION TITLE
Control how the Taxonomy box appears: Radio / Select / Dropdown etc

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1718] [Feature Requests]
### Discussed in https://github.com/TaxoPress/TaxoPress/discussions/1489

<div type='discussions-op-text'>

<sup>Originally posted by **stevejburge** April 23, 2021</sup>
By default, there's only a search box.

We could provide options such as radio buttons, or ways to limit the number of tags being added:
https://wordpress.org/plugins/radio-buttons-for-taxonomies/</div>

**Comment by ojopaul** [comment]
@stevejburge This is not possible with Block editor, at least, as at today.

**Comment by stevejburge** [comment]
@ojopaul I guess we'd need to provide our own metabox for this, right?

**Comment by ojopaul** [comment]
@stevejburge We can't remove Gutenberg sidebar for metabox and replace it with classic one. It'll break alot of other website features and those that expect the sidebar panels.

Good example of what we'll break is Checklists, Yoast SEO, RankMath just to mention few.

The feature will break almost every plugins that depends on the block sidebar panel for each Taxonomy.
